buildbot/master.cfg

```diff
@@ -126,38 +126,38 @@ def make_dolphin_win_build(build_type, arch, mode="normal"):
 
     f.addStep(GitNoBranch(repourl="https://github.com/dolphin-emu/dolphin.git",
                           progress=True, mode="incremental"))
-    f.addStep(RemoveDirectory(dir="build/Binary"))
+    f.addStep(RemoveDirectory(dir="build"))
+    f.addStep(MakeDirectory(dir="build"))
 
     branch = WithProperties("%s", "branchname")
     env = {"DOLPHIN_BRANCH": branch, "DOLPHIN_DISTRIBUTOR": "dolphin-emu.org"}
     if normal:
         env["DOLPHIN_DEFAULT_UPDATE_TRACK"] = "beta"
-    f.addStep(Compile(command=["msbuild.exe", "/v:m", "/p:Platform=%s" % msarch,
-                               "/p:Configuration=%s" % build_type,
-                               "dolphin-emu.sln"],
+    f.addStep(Compile(command=["cmake", "-GNinja", "-DCMAKE_BUILD_TYPE=%s" % (build_type), ".."],
+                      env=env, workdir="configuring",
```

Comment:
Not sure if you meant `workdir="configuring"` or `description="configuring"`. Seems odd to have a directory name ending in -ing.